libraries/AP_Notify/examples/ToshibaLED_test/ToshibaLED_test.cpp

```diff
@@ -21,7 +21,7 @@ void setup(void)
 
     // check if healthy
     if (!toshiba_led.healthy()) {
-        hal.console->printf("Failed to initialise Toshiba LED\n");
+        AP_HAL::panic("Failed to initialise Toshiba LED\n");
```

Comment:
I don't think this is a panic that should hold the whole system from booting.

Reply:
the examples are not at booting indeed.

Reply:
anyway, if we want to test it, why we need to continue if we don't have present a toshiba led on the board? The example especifically target to a Toshiba led, this example is not for generic test.